Content.Server/NPC/Systems/NPCSteeringSystem.cs

```diff
@@ -261,6 +261,8 @@ private void SetDirection(InputMoverComponent component, NPCSteeringComponent st
             EntityQuery<TransformComponent> xformQuery,
             float frameTime)
         {
+            IoCManager.InitThread(_dependencies, replaceExisting: true);
```

Comment:
I will pretend I didn't see this

Reply:
I saw pjb did it on gas tiles but I haven't historically had issues without it sooooooooo